### DISCUSSION TITLE
NationalMap v2.0 beta feedback thread

### DISCUSSION THREAD

**Discussion by hardreddata** [TerriaJS/terriajs#5108] [General]
I can't reorder layers?

https://nationalmap.prod.saas.terria.io/#share=s-uN150NcWbmKMCa6mvbrw1uUxpzf

That share link failed when I first tested it (?) but it is working now

I added

* Land Cover -> Geology -> Earthquake hazard 
* Elevation -> LiDAR 5 m DEM 

and realised I wanted the DEM below the hazard lines. 

I hope this helps. Thanks for the great work.

**Comment by hardreddata** [comment]
It is interesting to contrast the behavior of 

https://nationalmap.gov.au/#https://raw.githubusercontent.com/RussellGrew/nsw-covid-to-terria/master/catalog.json

and

https://nationalmap.prod.saas.terria.io/#https://raw.githubusercontent.com/RussellGrew/nsw-covid-to-terria/master/catalog.json

* The date on the beta is the start of the data and not the end of the data (latest).
* The nonlinear colour bar on the legacy does a better job with the "last 7 days cases" showing LGA other than Avalon with counts. This is after moving to the recent end of the timeline.

Credit for the layer goes to @kring and Data NSW. I forked it in order to maintain the data.

On "Next" as a whole I wonder whether the `tabbedCatalog` setting is done. Last I tried it on a clean clone each catalogue item was a tab.

**Reply by hardreddata** [reply]
Ok so `tabbedCatalog` is good. I just had to define some groups. The default `simple.json` doesn't have this.

**Comment by AnaBelgun** [comment]
Thank you @RussellGrew for the feedback. I'll submit a bug for the reordering issue.

**Comment by AnaBelgun** [comment]
@RussellGrew it looks like you can actually reorder the data but _only_ when you click on the text/name of the data. I've submitted examples in this ticket: https://github.com/TerriaJS/terriajs/issues/5111

**Reply by hardreddata** [reply]
Who knew you were so artistic with screenshot annotations? Thanks.
